source/index.html.md

```diff
@@ -67,175 +67,394 @@ Kittn expects for the API key to be included in all API requests to the server i
+                "name": "Sticker | Fnatic | Katowice 2014",
+                "scm": {
+                    "price": 41000,
+                    "volume": 0
+                }
+            },
+            {
+                "stickerId": 73,
+                "slot": 2,
+                "wear": 0.21217501,
+                "icon_url": "emskatowice2014/reason.d48f01f2758c2852ef32a68c49f7039ce211500a.png",
+                "name": "Sticker | Reason Gaming | Katowice 2014",
+                "scm": {
+                    "price": 118625,
+                    "volume": 0
+                }
+            },
+            {
+                "stickerId": 73,
+                "slot": 3,
+                "icon_url": "emskatowice2014/reason.d48f01f2758c2852ef32a68c49f7039ce211500a.png",
+                "name": "Sticker | Reason Gaming | Katowice 2014",
+                "scm": {
+                    "price": 118625,
+                    "volume": 0
+                }
+            }
+        ],
+        "tradable": 0,
+        "has_screenshot": true,
+        "scm": {
+            "price": 11,
+            "volume": 0
+        },
+        "item_name": "M4A1-S | Boreal Forest",
+        "wear_name": "Field-Tested",
+        "description": "It has been painted using a forest camouflage hydrographic.\\n\\n<i>The woods can be a dangerous place... never travel alone</i>",
+        "collection": "The Italy Collection",
+        "badges": []
+    },
+    "is_seller": false,
+    "min_offer_price": 7565,
+    "max_offer_discount": 1500,
+    "is_watchlisted": false,
+    "watchers": 0
 }
 ```
 
-This endpoint retrieves a specific kitten.
-
-<aside class="warning">Inside HTML code blocks like this one, you can't use Markdown, so use <code>&lt;code&gt;</code> blocks to denote code.</aside>
+This endpoint retrieves a specific listing.
 
 ### HTTP Request
 
-`GET http://example.com/kittens/<ID>`
+`GET https://csgofloat.com/api/v1/listings/<LISTING-ID>`
 
 ### URL Parameters
 
 Parameter | Description
 --------- | -----------
-ID | The ID of the kitten to retrieve
-
-## Delete a Specific Kitten
-
-```ruby
-require 'kittn'
-
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.delete(2)
-```
+listing_id | The ID of the listing to retrieve
 
-```python
-import kittn
-
-api = kittn.authorize('meowmeowmeow')
-api.kittens.delete(2)
-```
+## List an item
 
 ```shell
-curl "http://example.com/api/kittens/2" \
-  -X DELETE \
-  -H "Authorization: meowmeowmeow"
-```
-
-```javascript
-const kittn = require('kittn');
-
-let api = kittn.authorize('meowmeowmeow');
-let max = api.kittens.delete(2);
+curl -X POST "https://csgofloat.com/api/v1/listings" \
+-H "Authorization: <API-KEY>; Content-Type: application/json" \
+-d '{"asset_id": <ID>, "type": <TYPE>, ("price": <INT>,  "max_offer_discount": <INT> | "reserve_price": <INT>, "duration_days": <DURATION>), ["description": <TEXT>, "private": <BOOLEAN>]}'
```

Comment:
AFAIK, typically these shell examples provide examples of real inputs that you could do.

So you'd actually fill in type, asset_id, price, etc.. with example stuff

Probably a good idea to have two examples, one for auctions and one for buy now.